README.md

```diff
@@ -87,7 +87,7 @@ sbt assembly
 To test all test suites we can use:
 
 ```shell
-sbt test
+gradle test
 ```
 
 Testing an individual test suite can be specified by using `testOnly`. For example:
```

Comment:
testOnly is the sbt approach.
The Gradle approach is to use a test filter - but my quick experimentation has been unable to get this to work on this branch:

`gradle test --tests "uk.gov.ons.sbr.models.localunit.AddressSpec*"`

(tried with / without the * - and various other combinations)